examples/gcf-pubsub-vm-delete-event-handler/requirements.txt

```diff
@@ -0,0 +1,3 @@
+google-api-python-client==1.7.9
```

Comment:
do we need to pin all these dependencies?

Reply:
No need, just habit.  Will remove the version spec entirely, please comment if there's a more preferred way (e.g. specifying to major or minor version, but not bugfix).

Reply:
I typically recommend just specify a minimum version.

Reply:
Done